psi4/driver/procrouting/proc_table.py

```diff
@@ -245,6 +245,7 @@
     # Gradients
     if not (ssuper.is_c_hybrid() or ssuper.is_c_lrc() or ssuper.needs_vv10()):
         procedures['gradient'][key] = proc.run_scf_gradient
+        procedures['energy']['td-' + key] = proc.run_tdscf_energy
```

Comment:
just in case ppl pay more attention to the comment than the key, maybe move this out of `# Gradients`.